### PR TITLE
Add clear cache command

### PR DESCRIPTION
**Pull request by bpesch**

### CODE REVIEW

File: source/Internal/Framework/Theme/Command/ClearCacheCommand.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OxidEsales\EshopCommunity\Internal\Framework\Theme\Command;
```
Comment (alfredbez) [inline source/Internal/Framework/Theme/Command/ClearCacheCommand.php:5]: Maybe move it to a different namespace since it does not only clear the theme-cache files?

Comment (bpesch) [reply source/Internal/Framework/Theme/Command/ClearCacheCommand.php]: The container_cache.php is being generated into the tmp directory.

See:
https://github.com/OXID-eSales/oxideshop_ce/blob/master/source/Internal/Transition/Utility/BasicContext.php#L36

Comment (bpesch) [reply source/Internal/Framework/Theme/Command/ClearCacheCommand.php]: The command now supports cache folder clearing

File: tests/Integration/Internal/Framework/Module/Command/ActivateConfiguredModulesCommandTest.php [hunk 2]
```diff
@@ -134,15 +133,4 @@ private function prepareTestModuleConfigurations(bool $isConfigured, int $shopId
...
-    private function executeCommand(array $input): void
-    {
-        $app = $this->getApplication();
-
-        $this->execute(
-            $app,
-            $this->get('oxid_esales.console.commands_provider.services_commands_provider'),
-            new ArrayInput($input)
-        );
-    }
```
Comment (alfredbez) [inline tests/Integration/Internal/Framework/Module/Command/ActivateConfiguredModulesCommandTest.php:147]: Is this change related to the Pull-Request?

Comment (bpesch) [reply tests/Integration/Internal/Framework/Module/Command/ActivateConfiguredModulesCommandTest.php:147]: Yes, I wouldn't be able to use that function in the unit test otherwise.

**Comment by Sieg** [comment]
Hello guys, we have added this question to our backlog refinement, and most likely we will take it as a base for the final implementation. Thanks.

**Comment by JaroslavHerber** [comment]
It would be nice to have a button in /admin/ to clear the cache too.
